common/app/views/support/HtmlCleaner.scala

```diff
@@ -341,11 +341,12 @@ case class TagLinker(article: Article)(implicit val edition: Edition, implicit v
 
     if (article.content.showInRelated) {
 
-      // Get all paragraphs which are not contained in a pullquote
-      val paragraphs = doc.getElementsByTag("p").filterNot( p =>
-        p.parents.exists( ancestor =>
-          ancestor.tagName() == "aside" && ancestor.hasClass("element-pullquote")
-        )
+      // Get all paragraphs which are not contained in a pullquote or in an instagram caption
+      val paragraphs = doc.getElementsByTag("p").filterNot( p => p.parents.exists { ancestor =>
```

Comment:
I am nitpicking a lot here: I would keep `p.parents...` on a new line. Feel free to ignore me though and push as it is